include/caffe/layers/cudnn_conv_layer.hpp

```diff
@@ -65,7 +66,16 @@ class CuDNNConvolutionLayer : public ConvolutionLayer<Dtype> {
   size_t *workspace_bwd_data_sizes_;
   size_t *workspace_bwd_filter_sizes_;
   GPUMemoryManager::Buffer workspace;
-  int backward_passed_ctr_;
+
+ private:
+  bool use_algo_seeker_;
+  bool use_modest_workspace_;
+  void FindExConvAlgo(const vector<Blob<Dtype>*>& bottom,
+                    const vector<Blob<Dtype>*>& top,
+                    const size_t workspace_bytes);
+  void GetConvAlgo(const vector<Blob<Dtype>*>& bottom,
+                    const vector<Blob<Dtype>*>& top,
+                    const size_t workspace_bytes);
 };
```

Comment:
I disagree actually, I like 1 argument / line when they're this long. They should be aligned on the const though ;)